### PR TITLE
Add 1.6.4 release

### PR DESCRIPTION
**Pull request by sunjincheng121**
Add the Download message and add release notes  for release-1.6.4.

### CODE REVIEW

File: _posts/2019-02-15-release-1.6.4.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "Apache Flink 1.6.4 Released"
+date:   2019-02-15 12:00:00
+categories: news
+---
+
+The Apache Flink community released the fourth bugfix version of the Apache Flink 1.6 series.
+
+This release includes more than 60 fixes and minor improvements for Flink 1.6.3. The list below includes a detailed list of all fixes.
```
Comment (fhueske) [inline _posts/2019-02-15-release-1.6.4.md]: `60` -> `25` (according to the Jira release notes)?

**Comment by fhueske** [comment]
Thanks for working on the 1.6.4 release!
The blog post looks good (please check my comment).

However, we need to update the `_config.yml` file similar as it was done for the [1.6.3 announcement](https://github.com/apache/flink-web/commit/f7f2a8e0260ea641e55e0a6040da9b6c815929d0).

Thank you, Fabian

File: _posts/2019-02-15-release-1.6.4.md (hunk 1 above)
Comment (sunjincheng121) [reply _posts/2019-02-15-release-1.6.4.md]: +1

**Comment by sunjincheng121** [comment]
Thanks for help review the website changes, @fhueske  !
I have update the PR. according to your comment.
Best,
Jincheng

**Comment by wuchong** [comment]
Thanks @sunjincheng121 .  The changes look good to me. 

+1

**Comment by fhueske** [comment]
PR was merged.

**Pull request closed without merging**